src/api/login/login.service.ts

```
@@ -163,4 +154,115 @@ export class LoginService {
+        throw new CustomException(ERROR_CODES.GOOGLE_AUTH_FAILED);
+      }
+      return user;
+    }
+
+    const email = profile?.email ?? `google_${sid}@placeholder.local`;
+    const user = await this.userService.createUser(email);
+
+    await this.socialLoginService.createSocialLogin(user.id, sid, 'google');
+    return user;
+  }
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)